cfgov/cfgov/urls.py

```diff
@@ -406,6 +407,12 @@ def empty_200_response(request, *args, **kwargs):
         name='regulations3k-service-worker.js'
     ),
 
+    # These work:
+    url(r'^api/v2/', api_router.urls),
+    flagged_url('ENABLE_WAGTAIL_API', r'^api/v1/', empty_200_response),
+    # This doesn't work:
+    #flagged_url('ENABLE_WAGTAIL_API', r'^api/v2/', api_router.urls),
```

Comment:
When I uncomment this line, I get this traceback in the Django log: (@willbarton)

```
Unhandled exception in thread started by <function check_errors.<locals>.wrapper at 0x10b2b9488>
Traceback (most recent call last):
  File "/Users/lortonel/.virtualenvs/cfgov-refresh3.6/lib/python3.6/site-packages/django/utils/autoreload.py", line 228, in wrapper
    fn(*args, **kwargs)
  File "/Users/lortonel/.virtualenvs/cfgov-refresh3.6/lib/python3.6/site-packages/django/core/management/commands/runserver.py", line 124, in inner_run
    self.check(display_num_errors=True)
  File "/Users/lortonel/.virtualenvs/cfgov-refresh3.6/lib/python3.6/site-packages/django/core/management/base.py", line 359, in check
    include_deployment_checks=include_deployment_checks,
  File "/Users/lortonel/.virtualenvs/cfgov-refresh3.6/lib/python3.6/site-packages/django/core/management/base.py", line 346, in _run_checks
    return checks.run_checks(**kwargs)
  File "/Users/lortonel/.virtualenvs/cfgov-refresh3.6/lib/python3.6/site-packages/django/core/checks/registry.py", line 81, in run_checks
    new_errors = check(app_configs=app_configs)
  File "/Users/lortonel/.virtualenvs/cfgov-refresh3.6/lib/python3.6/site-packages/django/core/checks/urls.py", line 16, in check_url_config
    return check_resolver(resolver)
  File "/Users/lortonel/.virtualenvs/cfgov-refresh3.6/lib/python3.6/site-packages/django/core/checks/urls.py", line 26, in check_resolver
    return check_method()
  File "/Users/lortonel/.virtualenvs/cfgov-refresh3.6/lib/python3.6/site-packages/django/urls/resolvers.py", line 257, in check
    warnings.extend(check_resolver(pattern))
  File "/Users/lortonel/.virtualenvs/cfgov-refresh3.6/lib/python3.6/site-packages/django/core/checks/urls.py", line 26, in check_resolver
    return check_method()
  File "/Users/lortonel/.virtualenvs/cfgov-refresh3.6/lib/python3.6/site-packages/django/urls/resolvers.py", line 256, in check
    for pattern in self.url_patterns:
  File "/Users/lortonel/.virtualenvs/cfgov-refresh3.6/lib/python3.6/site-packages/flags/urls.py", line 75, in url_patterns
    pattern.default_args, pattern.name)
AttributeError: 'RegexURLResolver' object has no attribute 'default_args'
```